src/Components/Artwork/SaveButton/SaveButton.tsx

```diff
@@ -17,18 +18,28 @@ export interface SaveButtonProps {
 
 interface SaveButtonBaseProps {
   isSaved: boolean
+  artwork: SaveArtworkToListsButton_artwork$data | SaveButton_artwork$data
   onClick: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
 }
 
-const BTN_HEIGHT = 20
-const BTN_WIDTH = 20
+const BTN_HEIGHT = 18
+const BTN_WIDTH = 18
 
 export const SaveButtonBase: React.FC<SaveButtonBaseProps> = ({
   isSaved,
   onClick,
+  artwork,
 }) => {
   const [isHovered, setIsHovered] = useState(false)
   const title = isSaved ? "Unsave" : "Save"
+  const { lotWatcherCount, lotClosesAt, liveBiddingStarted } =
+    artwork.collectorSignals?.auction || {}
+
+  const shouldDisplayLotCount =
+    lotWatcherCount != null &&
+    lotWatcherCount > 0 &&
+    !liveBiddingStarted &&
+    (!lotClosesAt || new Date(lotClosesAt) >= new Date())
```

Comment:
Again here i think the count presence+non-zero could be simplified to `!!lotWatcherCount`

Bigger question I have here: Does this count update when the users saves/unsaves? As a user I would expect to see that change after i click the heart, but as a developer I would not expect to see that change reflected in the mutation result because it is processed in a job outside the mutation request.

3 ways I could see this working:
1. I could be wrong and it does just work.
2. We update the artwork store surgically when the save/unsave action happens
3. We just maintain some local state in this component including
  - the count on load (in this case i would avoid the null value altogether and set it like `const lotWatcherCount = artwork.collectorSignals.auction.lotWatcherCount || 0`)
  - whether the user had the artwork saved on load (so whether the count includes the user)
  - a state var for when the user saves/unsaves to +/-1